graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/processor/QueryMappingModel.java

```diff
@@ -72,6 +72,15 @@ public EntityMappingModel getEntity() {
       String parentTypeName,
       Map<String, EntityMappingModel> entities,
       ProcessingContext context) {
+    System.out.println(
+        "QueryMappingModel.build, query: "
+            + query
+            + " , parentTypeName: "
+            + parentTypeName
+            + " , entities: "
+            + entities
+            + "context:"
+            + context);
 
```

Comment:
As far as I can see, the QueryMappingModel is already taking all PKs (including Cks) into account by using this:
```
List<FieldMappingModel> primaryKey = entity.getPrimaryKey();
```
Are the only modifications that we need to do as a part of this ticket is:
- adding a way to pass CK in query and entity
- add a possibility to return a list from `QueryMappingModel` 
Is that true?

Reply:
Yes. Given the model:
```graphql
type Foo {
  pk1: Int! @cql_column(partitionKey: true)
  pk2: Int! @cql_column(partitionKey: true)
  cc1: Int! @cql_column(clusteringOrder: ASC)
  cc2: Int! @cql_column(clusteringOrder: DESC)
}
```

The only query that can be mapped right now is:
```graphql
foo(pk1: Int!, pk2: Int!, cc1: Int!, cc2: Int!): Foo
```
The goal is to add:
```graphql
foos(pk1: Int!, pk2: Int!, cc1: Int!): [Foo]
foos(pk1: Int!, pk2: Int!): [Foo]
```
Any prefix of the primary key should be allowed, as long as it's in order, and the full partition key is present (e.g. `foos(pk1: Int!)` is invalid).